Quadriga/src/main/java/edu/asu/spring/quadriga/dto/UploadTransfomationFilesDTO.java

```diff
@@ -0,0 +1,106 @@
+package edu.asu.spring.quadriga.dto;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.Column;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.io.Serializable;
+import javax.persistence.Id;
+
+/**
+ * DTO for holding the metadata of transformation files
+ * 
+ * @author JayaVenkat
+ *
+ */
+@Entity
+@Table(name = "tbl_transfomationfiles_metadata")
+@XmlRootElement
+public class UploadTransfomationFilesDTO implements Serializable {
+
+	private static final long serialVersionUID = 1L;
+	@Id
+	private String Id;
+	@Column(name = "Mapping_Title")
+	private String mappingTitle;
+	@Column(name = "Mapping_Description")
+	private String mappingDescription;
+	@Column(name = "MappingFile")
+	private String mappingFile;
+	@Column(name = "Transform_Title")
+	private String transfomationTitle;
+	@Column(name = "TransformDescription")
+	private String transformationDescription;
+	@Column(name = "Transform_File")
+	private String transformFile;
```

Comment:
I think we talked about that, right? That there should be a title and a description for the whole mapping (both files together)?